public/app/features/dashboard-scene/serialization/transformSceneToSaveModelSchemaV2.test.ts

```diff
@@ -525,6 +531,39 @@ describe('dynamic layouts', () => {
     expect(respGridLayout.items.length).toBe(2);
     expect(respGridLayout.items[0].kind).toBe('ResponsiveGridLayoutItem');
   });
+
+  it('should transform scene with tabs layout to schema v2', () => {
+    const tabs = [
+      new TabItem({
+        layout: new DefaultGridLayoutManager({
+          grid: new SceneGridLayout({
+            children: [
+              new DashboardGridItem({
+                y: 0,
+                height: 10,
+                body: new VizPanel({}),
+              }),
+            ],
+          }),
+        }),
+      }),
+    ];
+
+    const scene = setupDashboardScene(
+      getMinimalSceneState(
+        new TabsLayoutManager({
+          currentTab: tabs[0],
```

Comment:
Out of curiosity. Why don't we use the tab index here? This would be easier to serialize and avoid unnecessary references to an object living in an array.

Reply:
I'm not sure we want to serialize currentTab. That would be adding the state of the dashboard to the persisted model. So I think it's irrelevant for the schema. But yes, maybe it's worth rethinking why currentTab is not just an index in the layoutManager.

Reply:
It was easier for me to do it like this when developing it but we can change it to be an index.

`currentTab` won't be neither serialized nor persisted in the first implementation. We may rethink this but for the time being we always fallback to the first tab in the list.

Reply:
+1 to @ivanortegaalba comment, not sure what benefit it brings to have the tab instance here.